pro/src/ui-kit/form_rff/fields/SirenField/decorators.ts

```diff
@@ -13,9 +13,16 @@ export const sirenUpdate = async (humanSiren: string) => {
       name: '',
       postalCode: '',
       siren: humanSiren,
+      apeCode: '',
     }
   }
-  return response.payload.values
+  let values = undefined
+  if (response.payload.values != null) {
+    // eslint-disable-next-line @typescript-eslint/no-unused-vars
+    const { apeCode, ...rest } = response.payload.values
```

Comment:
```suggestion
    const { apeCode,  } = response.payload.values
```
hé hop, plus de ts ignore :)
Sinon beacoup de linter sont configurés pour ignorer les variable préfixé avec `_` (je ne crois pas que ca soit le cas chez nous).

Reply:
Sauf que le problème c'est le `apeCode`. 

Je veux m'en débarrasser et passer la variable `rest` dans la variable juste en dessus.

Reply:
ha oui, ca c'est mort ^^